src/drivers/code_hosting/gitlab.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+  local parent_branch=$3
+
+  local to_compare
+  if [ "$parent_branch" = "$MAIN_BRANCH_NAME" ]; then
+    # Allow GitLab to redirect to the proper place if this repository is a fork
+    to_compare="$branch"
```

Comment:
That's exactly what it is 😉 My thinking was that I learn exactly what changes are needed by running the tests and fixing the implementation until they passed. Thankfully very little needed to be changed to get the tests to pass.

I'm not clear what is returning a 404. The `open_browser` call on L17 is working fine for me.